src/frontend/src/stores/gridSnapping.ts

```diff
@@ -0,0 +1,11 @@
+import { create } from "zustand";
+import { GridStoreType } from "../types/zustand/grid";
+
+export const useGridSnappingStore = create<GridStoreType>((set, get) => ({
+  gridSnapping: JSON.parse(window.localStorage.getItem("isGridSnap")!) ?? false,
```

Comment:
hey, what do you think about this change
```
gridSnapping: window.localStorage.getItem("isGridSnap") ? JSON.parse(window.localStorage.getItem("isGridSnap") : false ,
```

Reply:
hi, what do you think about this one? we won't be calling 'getItem' twice
`gridSnapping: JSON.parse(window.localStorage.getItem("isGridSnap") ?? "false"),`

this version handles cases where the item doesn't exist in localStorage

in general what I had i took from 
/src/frontend/src/stores/darkStore.ts
`dark: JSON.parse(window.localStorage.getItem("isDark")!) ?? false,`

Reply:
its perfect! it can be that